clang/test/Driver/compress.c

```diff
@@ -1,5 +1,9 @@
+// XFAIL: system-aix
+
 // REQUIRES: zlib
 
+; XFAIL: target={{.*}}-aix{{.*}}
```

Comment:
You should use the C++ comment format here:
```suggestion
// XFAIL: target={{.*}}-aix{{.*}}
```